python/feathub/feature_views/tests/test_derived_feature_view.py

```diff
@@ -52,6 +52,7 @@ def test_derived_feature_view_filter_expr(self):
         expected_result_df = expected_result_df.sort_values(by=["time"]).reset_index(
             drop=True
         )
+        expected_result_df = expected_result_df[result_df.columns]
```

Comment:
We may not need to reorder the fields of `expected_result_df` according to the previous comments. And if we have to reorder the fields, it might be better to modify the initialization of `expected_result_df` directly.

Reply:
The `expected_result_df` is usually a copy of the input data, we don't initialize it. And we already have tests to ensure the order of feature view is correct, so we don't have to construct the `expected_result_df` with field names explicitly. If you're worrying about the risk of the different numbers of columns, I can check the number of columns before adjusting the order of columns. How do you think？